app/controllers/flipper_info.rb

```diff
@@ -0,0 +1,16 @@
+# Expose currently enabled features from Flipper for admin / dashboard requests
+module FlipperInfo
+  def self.included(base)
+    base.helper_method :current_enabled_features
+  end
+
+  def current_enabled_features
+    @current_enabled_features ||= Flipper.features.select { |feature| feature.enabled?(current_admin) }.map(&:name)
+  end
+
+  def set_enabled_features_as_datadog_tags
+    current_span = Datadog.tracer.active_span
+    return if current_span.nil?
+    current_span.set_tag(:enabled_features, current_enabled_features)
```

Comment:
Do we need to set individual tags for each flag, to make it easier to filter in datadog? If not, then ignore me.

Reply:
I dug around into datadog docs and some of their PRs, and its a bit confusing because:

* datadog tags are _sent_ as flat key/value pairs -- you can send other things from ruby-land, but they serialized to strings.
* datadog tags are represented in the UI as something smarter - for example, you'll notice there are tags like `http.method`, which is shown in the DD UI as a `http` hash where you can drill into the key values underneath http.  So you can setup searches or filters on `http.method` for example.

So for now I'm going to merge this and then see how it looks and filters in the Datadog UI, and if we need to build up some sort of smarter tag structure for this sort of thing we can do it.

Related docs: https://github.com/DataDog/dd-trace-rb/pull/1081 and https://docs.datadoghq.com/getting_started/tagging/using_tags?tab=assignment